routes/api.php

```diff
@@ -1,14 +1,27 @@
 <?php
 
 use Illuminate\Support\Facades\Route;
+
 use App\Http\Controllers\CreateCityController;
 use App\Http\Controllers\DestroyCityController;
 use App\Http\Controllers\GetAllCitiesController;
 use App\Http\Controllers\UpdateCityController;
 
+use App\Http\Controllers\Airlines\GetAllAirlinesController;
+use App\Http\Controllers\Airlines\StoreAirlineController;
+use App\Http\Controllers\Airlines\DestroyAirlineController;
+use App\Http\Controllers\Airlines\UpdateAirlineController;
+
 Route::prefix('cities')->name('cities.')->group(function () {
     Route::get('/', GetAllCitiesController::class)->name('index');
     Route::post('/', CreateCityController::class)->name('store');
-    Route::put('/{id}', UpdateCityController::class)->name('update');
+    Route::patch('/{id}', UpdateCityController::class)->name('update');
     Route::delete('/{id}', DestroyCityController::class)->name('destroy');
+});
+
+Route::prefix('airlines')->name('airlines.')->group(function () {
+    Route::get('/', GetAllAirlinesController::class)->name('index');
+    Route::post('/', StoreAirlineController::class)->name('store');
+    Route::patch('/{id}', UpdateAirlineController::class)->name('update');
+    Route::delete('/{id}', DestroyAirlineController::class)->name('destroy');
```

Comment:
```suggestion
    Route::put('/{airline}', UpdateAirlineController::class)->name('update');
    Route::delete('/{airline}', DestroyAirlineController::class)->name('destroy');
```
I highly suggest you use [Route Model Binding](https://laravel.com/docs/11.x/routing#route-model-binding). Unless you strictly need the `id` of a resource, we always use Route Model Binding.

Reply:
This was marked as resolved, tho it wasn't 👀

Reply:
I did change those routes! I did not change the city routes, because I considered that it would be incorrect to do it in this PR

Reply:
You're totally right. I got confused with the city routes. Sorry 🙏🏻